fonts.json

```
@@ -1866,6 +1866,21 @@
         "year": 2023,
         "zerostyle": "slashed"
     },
+    "paper-mono": {
+        "author": "Lost Coast Labs, Inc. (Paper Design)",
+        "description": "A beautiful monospace font for design and code.",
```

```suggestion
        "description": "A beautiful monospace font for design and code. Based on Geist Mono.",
```

I’d like to mention that, helps people discover more fonts.